main.go

```diff
@@ -75,6 +75,21 @@ func changeBook(bookName string) error {
 		return err
 	}
 
+	// Now add this book to the .dnote file, for issue #2
+	dnote, err := utils.GetDnote()
+	if err != nil {
+		return err
```

Comment:
I'm wondering, does this work the same as in Ruby where you can use the contents of the statement as a more of a guard clause and do something along the lines of: `return err if err != nil` ?